ts/features/bonus/bpd/screens/optInPaymentMethods/OptInPaymentMethodsCashbackUpdateScreen.tsx

```diff
@@ -1,36 +1,134 @@
 
-const OptInPaymentMethodsCashbackUpdateScreen = () => (
-  // The void customRightIcon and customGoBack are needed to have a centered header title
-  <BaseScreenComponent
-    showInstabugChat={false}
-    goBack={false}
-    headerTitle={I18n.t("bonus.bpd.optInPaymentMethods.cashbackUpdate.header")}
-    customRightIcon={{
-      iconName: "",
-      onPress: () => true
-    }}
-    customGoBack={
-      <ButtonDefaultOpacity onPress={() => true} transparent={true} />
+const styles = StyleSheet.create({
+  logo: {
+    resizeMode: "contain",
+    width: 48,
+    height: 46
+  },
+  headerContainer: {
+    ...IOStyles.row,
+    justifyContent: "space-between"
+  }
+});
+const OptInPaymentMethodsCashbackUpdateScreen = () => {
+  const navigation = useNavigationContext();
+  const dispatch = useIODispatch();
+  const bpdPaymentMethods = useIOSelector(getBpdAMethodsSelector);
+  const paymentMethods = useIOSelector(paymentMethodsSelector);
+  const bpdInfo = useIOSelector(availableBonusTypesSelectorFromId(ID_BPD_TYPE));
+  const bpdLogo: ImageSourcePropType = bpdInfo?.cover
+    ? { uri: bpdInfo?.cover }
+    : dafaultLogo;
+
+  // This screen should be shown only if the payment method are correctly loaded
+  if (paymentMethods.kind !== "PotSome") {
+    dispatch(optInPaymentMethodsFailure("error in the payment method loading"));
+  }
+
+  const handleOnContinuePress = () => {
+    if (bpdPaymentMethods.length > 0) {
+      navigation.navigate(navigateToOptInPaymentMethodsChoiceScreen());
+    } else {
+      dispatch(optInPaymentMethodsCompleted());
     }
-  >
-    <SafeAreaView
-      style={IOStyles.flex}
-      testID={"OptInPaymentMethodsCashbackUpdate"}
+  };
+
+  return (
+    // The void customRightIcon and customGoBack are needed to have a centered header title
+    <BaseScreenComponent
+      showInstabugChat={false}
+      goBack={false}
+      headerTitle={I18n.t(
+        "bonus.bpd.optInPaymentMethods.cashbackUpdate.header"
+      )}
+      customRightIcon={{
+        iconName: "",
+        onPress: () => true
+      }}
+      customGoBack={
+        <ButtonDefaultOpacity onPress={() => true} transparent={true} />
+      }
     >
-      <ScrollView style={[IOStyles.horizontalContentPadding]}>
-        <H1>{I18n.t("bonus.bpd.optInPaymentMethods.cashbackUpdate.title")}</H1>
-        <View spacer />
-      </ScrollView>
-    </SafeAreaView>
-  </BaseScreenComponent>
-);
+      <SafeAreaView
+        style={IOStyles.flex}
+        testID={"OptInPaymentMethodsCashbackUpdate"}
+      >
+        <ScrollView style={[IOStyles.horizontalContentPadding]}>
+          <View style={styles.headerContainer}>
+            <View>
+              <H2>
+                {I18n.t("bonus.bpd.optInPaymentMethods.cashbackUpdate.title")}
+              </H2>
+              <Text>
+                {I18n.t(
+                  "bonus.bpd.optInPaymentMethods.cashbackUpdate.subtitle"
+                )}
+              </Text>
+            </View>
```

Comment:
```suggestion
            <View style={IOStyles.flex}>
              <H2>
                {I18n.t("bonus.bpd.optInPaymentMethods.cashbackUpdate.title")}
              </H2>
              <Text>
                {I18n.t(
                  "bonus.bpd.optInPaymentMethods.cashbackUpdate.subtitle"
                )}
              </Text>
            </View>
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏A suggestion to avoid the image overflow on small devices:

Before            |  After
:-------------------------:|:-------------------------:
<img width="250" alt="Schermata 2022-02-07 alle 12 17 18" src="https://user-images.githubusercontent.com/26501317/152778248-26d5afed-3419-4f01-97ab-83b7f0c2b4f0.png"> | <img width="250" alt="Schermata 2022-02-07 alle 12 17 41" src="https://user-images.githubusercontent.com/26501317/152778322-d1934f9d-54a9-428d-8f1c-41950ae15f6f.png">

Reply:
Thank you, fixed in [3544594](https://github.com/pagopa/io-app/pull/3725/commits/35445945d4151995f692f944336b5dc7d1e42ea9)